app/screens/navigation.ts

```diff
@@ -499,6 +499,11 @@ export function goToScreen(name: AvailableScreens, title: string, passProps = {}
 
     DeviceEventEmitter.emit(Events.TAB_BAR_VISIBLE, false);
 
+    if (NavigationStore.getScreensInStack().includes(name)) {
```

Comment:
I do not remember if screen in stacks include modals and overlays, can you double check that?

Reply:
Yes, they include modals and overlays. I just checked it, and it includes `Permalink` when pressed to open a channel option from the thread view's options.

Reply:
If that is the case, when you pop to a screen that is a modal, the permalink etc.. does it work the same? I ask cause this is really to push a screen, so I wonder if that would have potential erratic behavior

Reply:
Ok sounds good, if that ends up being a case then will fix it eventually :)